### DISCUSSION TITLE
matlab-yalmip+cplex warning

### DISCUSSION THREAD

**Discussion by aswdjkl** [yalmip/YALMIP#1209] [General discussion]
matlab-yalmip+cplex 
appear
"Using Matlab Toolbox parameters in the CPLEX Matlab API is deprecated.They will be treated as unknown parameters in the future. "
how do i solve it?
help  thanks
my English is poor

**Comment by johanlofberg** [comment]
Nothing. It simply says cplex will behave differently in the future so YALMIP will be updated (but I think cplex is dropping matlab support so your better approach is to switch to another soler)

**Reply by aswdjkl** [reply]
ok,thanks for your help.
